codec/encoder.go

```diff
@@ -44,7 +44,7 @@ func (e *encoder) Length() int {
 		return len(e.cache)
 	}
 
-	e.Encode()
+	_, _ = e.Encode()
```

Comment:
you might want to explain through a comment why you explicitly decided to discard errors

Reply:
it's explained in GoDoc at line 40, do you want something closer to the code?